backend/src/graphql/resolvers/TableResolver.spec.ts

```diff
@@ -2159,6 +2159,109 @@ describe('TableResolver', () => {
           })
         })
       })
+
+      describe('private meeting in DB', () => {
+        beforeAll(async () => {
+          const meeting = await prisma.meeting.create({
+            data: {
+              name: 'Dreammall Entwicklung',
+              meetingID: 'Dreammall-Entwicklung-2',
+              attendeePW: '1234',
+              public: false,
+            },
+          })
+          await prisma.usersInMeetings.create({
+            data: {
+              meetingId: meeting.id,
+              userId: user.id,
+            },
+          })
+        })
+
+        beforeEach(() => {
+          getMeetingsMock.mockResolvedValue([
+            {
+              meetingName: 'Dreammall Entwicklung',
+              meetingID: 'Dreammall-Entwicklung-2',
+              internalMeetingID: '258ea7269760758304b6b8494f17e9bf69dc1efe-1718189921310',
+              createTime: 1718189921310,
+              createDate: new Date('Wed Jun 12 10:58:41 UTC 2024'),
+              voiceBridge: 96378,
+              dialNumber: '613-555-1234',
+              attendeePW: 'MqgUFwdD',
+              moderatorPW: 'mTtxYGo2',
+              running: true,
+              duration: 0,
+              hasUserJoined: true,
+              recording: false,
+              hasBeenForciblyEnded: false,
+              startTime: 1718189,
+              endTime: 0,
+              participantCount: 0,
+              listenerCount: 1,
+              voiceParticipantCount: 0,
+              videoCount: 0,
+              maxUsers: 0,
+              moderatorCount: 1,
+              attendees: {
+                attendee: {
+                  userID: '1234',
+                  fullName: 'Peter Lustig',
+                  role: 'moderator',
+                  isPresenter: false,
+                  isListeningOnly: false,
+                  hasJoinedVoice: true,
+                  hasVideo: true,
+                  clientType: 'html5',
+                },
+              },
+              metadata: '',
+              isBreakout: false,
+            },
+          ])
+        })
+
+        afterAll(async () => {
+          await prisma.usersInMeetings.deleteMany({
+            where: {
+              userId: user.id,
+            },
+          })
+          await prisma.meeting.deleteMany({
+            where: {
+              meetingID: 'Dreammall-Entwicklung-2',
+            },
+          })
+        })
+
+        it('returns empty array of meeting', async () => {
+          jest.clearAllMocks()
+          await expect(
+            testServer.executeOperation(
+              {
+                query:
+                  'query { openTables { id meetingName meetingID participantCount startTime attendees { fullName } } }',
+              },
+              {
+                contextValue: {
+                  user: bibiUser,
+                  dataSources: { prisma },
+                },
```

Comment:
Could we have another test for:

```suggestion
                contextValue: {
                  user,
                  dataSources: { prisma },
                },
```


I see that e.g. `getMeetingsMock` is mocked only for this case and we do not re-use:
```ts
const query = 'query { openTables { id meetingName meetingID participantCount startTime attendees { fullName } } }'
```